package/gluon-status-page/files/lib/gluon/status-page/www/index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="en">
 	<head>
-		<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate">
-		<meta http-equiv="Pragma" content="no-cache">
-		<meta http-equiv="Expires" content="0">
+		<meta charset="utf-8">
 		<meta http-equiv="refresh" content="0; URL=/cgi-bin/status">
+		<title>Redirect</title>
 	</head>
 	<body>
+	Redirect to <a href="/cgi-bin/status">/cgi-bin/status</a>
```

Comment:
I think showing such a link for a moment is very ugly, I'd rather leave the body empty.